tests/src/python/test_qgsdatumtransforms.py

```diff
@@ -246,6 +246,45 @@ def testNoLasLos(self):
         self.assertTrue(ops[1].name)
         self.assertTrue(ops[1].proj)
 
+    @unittest.skipIf(QgsProjUtils.projVersionMajor() < 8, 'Not a proj >= 8 build')
+    def testDatumEnsembles(self):
+        """
+        Test datum ensemble details
+        """
+        crs = QgsCoordinateReferenceSystem()
+        self.assertFalse(crs.datumEnsemble().isValid())
+        self.assertEqual(str(crs.datumEnsemble()), '<QgsDatumEnsemble: invalid>')
+        crs = QgsCoordinateReferenceSystem('EPSG:3111')
+        self.assertFalse(crs.datumEnsemble().isValid())
+
+        crs = QgsCoordinateReferenceSystem('EPSG:3857')
+        self.assertTrue(crs.datumEnsemble().isValid())
```

Comment:
given the number of calls to crs.datumEnsemble(), might be better to store it into a variable